data/sql/updates/pending_db_world/rev_1601870525065182700.sql

```diff
@@ -0,0 +1,436 @@
+(-137508, 0, 3, 0, 34, 0, 100, 0, 0, 1, 0, 0, 0, 66, 0, 0, 0, 0, 0, 0, 8, 0, 0, 0, 0, 0, 0, 0, 6.2797, 'Kirin Tor Battle-Mage - On Reached Point 1 - Set Orientation 6.2797'),
+(-137508, 0, 4, 0, 60, 0, 100, 1, 0, 0, 0, 0, 0, 62, 603, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, -673.399, -87.85, 426.51, 6.2797, 'Kirin Tor Battle-Mage - On Update - Teleport (No Repeat)');
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = -137507);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(-137507, 0, 0, 0, 1, 0, 100, 1, 500, 500, 500, 500, 0, 11, 48310, 0, 0, 0, 0, 0, 10, 137510, 33779, 0, 0, 0, 0, 0, 0, 'Kirin Tor Battle-Mage - Out of Combat - Cast \'Transitus Shield Beam\' (No Repeat)'),
+(-137507, 0, 1, 2, 38, 0, 100, 0, 0, 1, 0, 0, 0, 69, 1, 0, 0, 0, 0, 0, 8, 0, 0, 0, 0, -674.524, -2.496, 425.95, 0.042202, 'Kirin Tor Battle-Mage - On Data Set 0 1 - Move To Position'),
+(-137507, 0, 2, 0, 61, 0, 100, 0, 0, 1, 0, 0, 0, 41, 0, 0, 0, 0, 0, 0, 10, 137510, 33779, 0, 0, 0, 0, 0, 0, 'Kirin Tor Battle-Mage - On Data Set 0 1 - Despawn Instant'),
+(-137507, 0, 3, 0, 34, 0, 100, 0, 0, 1, 0, 0, 0, 66, 0, 0, 0, 0, 0, 0, 8, 0, 0, 0, 0, 0, 0, 0, 6.2797, 'Kirin Tor Battle-Mage - On Reached Point 1 - Set Orientation 6.2797'),
+(-137507, 0, 4, 0, 60, 0, 100, 1, 0, 0, 0, 0, 0, 62, 603, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, -674.524, -2.496, 425.95, 6.2797, 'Kirin Tor Battle-Mage - On Update - Teleport (No Repeat)');
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = -136524);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(-136524, 0, 0, 0, 38, 0, 100, 0, 1, 1, 0, 0, 0, 29, 1, 135, 0, 0, 0, 0, 10, 136281, 33624, 0, 0, 0, 0, 0, 0, 'Kirin Tor Battle-Mage - On Data Set 1 1 - Start Follow Closest Creature \'Archmage Pentarus\''),
+(-136524, 0, 1, 0, 60, 0, 100, 1, 0, 0, 0, 0, 0, 62, 603, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, -679.573, -82.066, 427.233, 0.054839, 'Kirin Tor Battle-Mage - On Update - Teleport (No Repeat)');
+
+DELETE FROM `conditions` WHERE (`SourceEntry` = -136524);
```

Comment:
missing `SourceTypeOrReferenceId` and `SourceGroup` from `DELETE` clause, this might delete unwanted conditions